src/Resource/utils.ts

```diff
@@ -23,6 +23,8 @@ import { SparqlViewQueryResponse } from '../View/SparqlView/types';
 import { getSparqlView } from '../View/utils';
 import Store from '../utils/Store';
 
+const FILES_PATH_LABEL = 'files';
```

Comment:
what is the selfUrl is `http://api.v1/resources/org/label/_/I-love-files-but-i-am-not-one-of-them`?

Reply:
oh no!

Reply:
I added a test to describe this scenario